### PR TITLE
Add Amharic locale

### PR DESCRIPTION
**Pull request by SamAsEnd**
**Amharic** is the second most widely spoken Semitic language in the world (after Arabic).

The language serves as the official working language of the Ethiopian federal government, and is also the official or working language of several of Ethiopia's federal regions.

### CODE REVIEW

**Comment by rikschennink** [comment]
Thank you 🙏 When you're ready I'll merge.

File: locale/am-et.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    labelFileProcessingAborted: 'ፋይሉን መጫን ተጨናግፏል',
+    labelFileProcessingError: 'ፋይሉን በመጫን ላይ ችግር ተፈጥሯል',
+    labelFileProcessingRevertError: 'ፈይሉን በመቀልበስ ላይ ችግር ተፈጥሯል',
+    labelFileRemoveError: 'በማጥፋት ላይ ችግር ተፈጥሯል',
+    labelTapToCancel: 'ለማጨናገፍ ታፕ ያድርጉ',
+    labelTapToRetry: 'ደግሞ ለመሞከር ታፕ ያድርጉ',
+    labelTapToUndo: 'ወደነበረበት ለመመለስ ታፕ ያድርጉ',
+    labelButtonRemoveItem: 'ላጥፋ',
+    labelButtonAbortItemLoad: 'ላጨናግፍ',
+    labelButtonRetryItemLoad: 'ደግመሜ ልሞክር',
```
Comment (mickeymgk) [inline locale/am-et.js:19]:
```suggestion
    labelButtonRetryItemLoad: 'ደግሜ ልሞክር',
```

File: locale/am-et.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    labelButtonAbortItemProcessing: 'ይቅር',
+    labelButtonUndoItemProcessing: 'ወደነበረበት ልመልስ',
+    labelButtonRetryItemProcessing: 'ደግመሜ ልሞክር',
```
Comment (mickeymgk) [inline locale/am-et.js:22]:
```suggestion
    labelButtonRetryItemProcessing: 'ደግሜ ልሞክር',
```

File: locale/am-et.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    labelFileLoadError: 'በማምበብ ላይ ችግር ተፈጥሯል',
+    labelFileProcessing: 'ፋይሉን በመጫን ላይ',
+    labelFileProcessingComplete: 'ፋይሉን መጫን ተጠናቅቋል',
+    labelFileProcessingAborted: 'ፋይሉን መጫን ተጨናግፏል',
+    labelFileProcessingError: 'ፋይሉን በመጫን ላይ ችግር ተፈጥሯል',
+    labelFileProcessingRevertError: 'ፈይሉን በመቀልበስ ላይ ችግር ተፈጥሯል',
+    labelFileRemoveError: 'በማጥፋት ላይ ችግር ተፈጥሯል',
+    labelTapToCancel: 'ለማጨናገፍ ታፕ ያድርጉ',
+    labelTapToRetry: 'ደግሞ ለመሞከር ታፕ ያድርጉ',
+    labelTapToUndo: 'ወደነበረበት ለመመለስ ታፕ ያድርጉ',
```
Comment (mickeymgk) [inline locale/am-et.js:16]:
```suggestion
    labelTapToUndo: 'ወደነበረበት ለመመለስ ነካ ያድርጉ',
```

**Review by nebiyu-ethio — COMMENTED** [review]
It is 'በማንበብ ላይ', not 'በማምበብ ላይ'.
And 'በማንበብ ላይ ችግር ተፈጥሯል' not 'በማምበብ ላይ ችግር ተፈጥሯል'.

**Review by mickeymgk — CHANGES_REQUESTED** [review]

File: locale/am-et.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    labelButtonRemoveItem: 'ላጥፋ',
+    labelButtonAbortItemLoad: 'ላጨናግፍ',
```
[the remark this answers is not in the document]
Comment (Mahiamk) [reply locale/am-et.js:18]: Line 16 you jumped Without changing

**Comment by SamAsEnd** [comment]
It's ready

**Pull request merged**